src/main/kotlin/org/rust/ide/annotator/RsCargoCheckAnnotator.kt

```diff
@@ -147,18 +147,23 @@ private fun filterMessage(file: PsiFile, document: Document, message: RustcMessa
     val spanFilePath = PathUtil.toSystemIndependentName(span.file_name)
     if (!file.virtualFile.path.endsWith(spanFilePath)) return null
 
+    @Suppress("NAME_SHADOWING")
     fun toOffset(line: Int, column: Int): Int? {
+        val line = line - 1
+        val column = column - 1
```

Comment:
Am I the only one who wishes every language had Haskell's treating `'` as valid identifier character, so it is possible to name variables `line'`, `column'` ?